Source/CombatExtended/Harmony/Harmony-SmokepopBelt.cs

```diff
@@ -26,8 +26,8 @@ public static bool Prefix(SmokepopBelt __instance, DamageInfo dinfo)
                 float statValue = __instance.GetStatValue(StatDefOf.SmokepopBeltRadius, true);
                 DamageDef smoke = DamageDefOf.Smoke;
                 Thing instigator = null;
-                ThingDef gas_Smoke = ThingDefOf.Gas_Smoke;
-                GenExplosion.DoExplosion(position, map, statValue, smoke, instigator, -1, null, null, null, gas_Smoke, 1f, 1, false, null, 0f, 1, 0f, false);
+                ThingDef gas_Smoke = ThingDefOf.Gas_Smoke;                             //TODO: postExplosionSpawnThingdDef? \|/ (gas_Smoke ?)
```

Comment:
This just mirrors vanilla smokepop, no additions necessary.

Reply:
Okay removing TODO.